src/streamlink/plugins/ntv.py

```diff
@@ -0,0 +1,23 @@
+import re
+from streamlink.plugin import Plugin
+from streamlink.stream import HLSStream
+
+
+class NTV(Plugin):
+    url_re = re.compile(r'https?://www.ntv.ru/air/.*')
+
+    @classmethod
+    def can_handle_url(cls, url):
+        return cls.url_re.match(url) is not None
+
+    def _get_streams(self):
+        body = self.session.http.get(self.url).text
+        mrl_re = re.compile(r'var camHlsURL = \'(.*)\'')
```

Comment:
You could make the regex more generic and use the same one for both, and only compile it one. If you only use a regex once, there is no need to compile it :)

This would probably serve for both. 
```python
mrl_re = re.compile(r'var (?:camH|h)lsURL = \'(.*)\'')
```

Reply:
In fact, you could just do

```python
body = self.session.http.get(self.url).text
match = re.search(r'var (?:camH|h)lsURL = \'(.*)\'', body)
if match:
  mrl = match.group(1)
  return HLSStream.parse_variant_playlist(self.session, mrl)
```

And it would match both just fine - obviously you need to test it, cos I just wrote it directly in the comments...

Reply:
I had the same idea before writing the code for last commit. I like compact and simple code and that was reason for exec () that You fairly criticized.  But...  In this case problem is that both pattern will match  in the same request (body) for any channels other than main one. Thus merging both pattern is not enough. Further we need additional code to distinguish requested channel. This requires more complex code than actual one. So I prefer this solution.

Reply:
I undertand :)

I think you add some more error handling, what if neither regex matches? Also, you can avoid performing the regex twice.

```python
    def _get_streams(self):
        body = self.session.http.get(self.url).text
        mrl = None
        match = re.search(r'var camHlsURL = \'(.*)\'', body)
        if match:
            mrl = 'http:' + match.group(1)
        else:
            match = re.search(r'var hlsURL = \'(.*)\'', body)
			if match:
	            mrl = match.group(1)
		if mrl:
	        return HLSStream.parse_variant_playlist(self.session, mrl)
```

Reply:
Done. Thanks, @beardypig.

Reply:
Are there something missing to get merging  completed?